src/lib/dnssd/Resolver.h

```diff
@@ -268,10 +268,14 @@ class Resolver
     virtual void Shutdown() = 0;
 
     /**
-     * Registers a resolver delegate. If nullptr is passed, the previously registered delegate
-     * is unregistered.
+     * Registers a resolver delegate.
```

Comment:
What is a resolver delegate in this context?

It appears all delegates will get all results of all queries on a given fabric, not just the ones they initiated.

Reply:
This can be handled in a separate PR, given this is just updating a comment, @vivien-apple feel free to resolve or update.